upgrade.sh

```diff
@@ -26,9 +27,10 @@ case $step in
     sudo systemctl stop nabd.socket || echo -n ""
     sudo systemctl stop nabd.service || echo -n ""
   
+    echo "Updating Pynab"
     sudo -u ${owner} touch /tmp/pynab.upgrade
     sudo chown ${owner} /tmp/pynab.upgrade
-    echo "Updating code - 1/?" > /tmp/pynab.upgrade
+    echo "Updating Pynab - 1/?" > /tmp/pynab.upgrade
```

Comment:
It doesn't really make sense to have the log going both to /tmp/pynab.upgrade and to stdout, duplicating lines. We should at least use tee if this is what we really mean.

Reply:
Stdout logging (without the "- x/y" progress mark) is intended for users that execute the script by hand.
But feel free to use `tee` if you wish.